corehq/apps/style/templates/style/includes/global_navigation_bar.html

```diff
@@ -56,6 +56,15 @@
                     <i class="fa fa-dashboard icon-dashboard dropdown-icon"></i> {% trans "Current Subscription" %}
                 </a>
             </li>
+            {% if request.couch_user_is_super_user %}
```

Comment:
removed it https://github.com/dimagi/commcare-hq/pull/13564/commits/4678473273844271b96dc45ff5d97125c1a142c5